components/Element/ChatbotFiles.vue

```diff
@@ -35,20 +42,20 @@ const checkUploadAllowed = (newFiles: any) => {
   return parseInt(useRuntimeConfig().public.demo.maxChatFiles) > num;
 };
 
-const files = ref([]);
+const files = ref<singleFile[]>([]);
 let indexedItems: any = [];
 const integration = useIntegration();
 
 const loadFiles = async () => {
   isLoading.value = true;
   const endpoint = `/api/${integration}/index/${collection?.uuid}/documents_metadata?filter[type]=files&sort=-created`;
   if (integration === 'brevia') {
-    const items = await $fetch(endpoint);
+    const items = await $fetch<singleFile[]>(endpoint);
     files.value = items;
     indexedItems = items;
   } else {
     const items = await useApiGetAll(endpoint);
-    files.value = items.data;
+    files.value = items.res;
```

Comment:
I would instead fix `composables/use-api-get-all.ts` with 

```ts
  return { data: res };
```

and leave `items.res` here and in other files